zerver/lib/actions.py

```diff
@@ -1812,6 +1812,24 @@ def recipient_for_emails(emails: Iterable[str], not_forged_mirror_message: bool,
         sender=sender
     )
 
+def recipient_for_user_ids(user_ids: Iterable[int], not_forged_mirror_message: bool,
+                           forwarder_user_profile: Optional[UserProfile],
+                           sender: UserProfile) -> Recipient:
+    user_profiles = []  # type: List[UserProfile]
+    for user_id in user_ids:
+        try:
+            user_profile = get_user_profile_by_id_in_realm(user_id, sender.realm)
+        except UserProfile.DoesNotExist:
+            raise ValidationError(_("Invalid user ID {}".format(user_id)))
+        user_profiles.append(user_profile)
+
+    return recipient_for_user_profiles(
+        user_profiles=user_profiles,
+        not_forged_mirror_message=not_forged_mirror_message,
```

Comment:
I don't think we need `not_forged_mirror_message` or `forwarded_user_profile` here, since typing indicators are Zulip-specific and not used in "mirror" scenarios.